lib/apiservers/engine/backends/container.go

```diff
@@ -205,8 +205,13 @@ func (c *ContainerBackend) ContainerExecCreate(name string, config *types.ExecCo
 
 	var eid string
 	operation := func() error {
-		if vc.TryLock(APITimeout) {
+		op.Infof("ContainerExecCreate trying to lock vc: %s", name)
```

Comment:
Can we drop this and the matching acquisition to Debug?
The else case we could leave as Info if you think that's worthy of logging by default.

Reply:
There are message in the retry package but they don't have the opID. I think we should add it but it touches many other files and I would like to do in another check-in. For now I am changing all message to Debug level.